Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -679,6 +750,7 @@ public protocol ProtobufBinaryMessageBase: ProtobufMessageBase {
     // Serialize to protobuf
     func serializeProtobuf() throws -> Data
     func serializeProtobufBytes() throws -> [UInt8]
```

Comment:
It's easy enough to modify the performance harness to try each one, so I'll do that as part of #44 (which I filed to ask the very same question 😄). My hope is that `Data` is the better one because it's what other file/stream APIs are going to return to clients, and since `Data` to `[UInt8]` conversion is very costly, they would have to stay as two separate code paths anyway.